Packs/SymantecEndpointSecurity/Integrations/SymantecEndpointSecurity/SymantecEndpointSecurity.py

```diff
@@ -0,0 +1,481 @@
+import itertools
+import demistomock as demisto
+from urllib3 import disable_warnings
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+from datetime import datetime
+import dateparser
+import time
+
+disable_warnings()
+
+
+# CONSTANTS
+VENDOR = "symantec"
+PRODUCT = "endpoint_security"
+DEFAULT_CONNECTION_TIMEOUT = 30
+MAX_CHUNK_SIZE_TO_READ = 1024 * 1024 * 150  # 150 MB
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+# Sleep time between fetch attempts when an error occurs in the retrieval process,
+# primarily used to avoid overloading with consecutive API calls
+# if an error is received from Symantec's side.
+FETCH_INTERVAL = 60
+
+
+class UnauthorizedToken(Exception):
+    """
+    Exception raised when the authentication token is unauthorized.
+    """
+
+    ...
+
+
+class NextPointingNotAvailable(Exception):
+    """
+    Exception raised when the next pointing is not available.
+    """
+
+    ...
+
+
+class Client(BaseClient):
+    def __init__(
+        self,
+        base_url: str,
+        token: str,
+        stream_id: str,
+        channel_id: str,
+        verify: bool,
+        proxy: bool,
+    ) -> None:
+
+        self.headers: dict[str, str] = {}
+        self.token = token
+        self.stream_id = stream_id
+        self.channel_id = channel_id
+        self.fetch_interval = FETCH_INTERVAL
+
+        super().__init__(
+            base_url=base_url,
+            verify=verify,
+            proxy=proxy,
+            timeout=180,
```

Comment:
why 180?

Reply:
I did this in another Symantec integration
this is a recommendation from their side
probably due to server timeouts on their side